.travis.yml

```diff
@@ -1,28 +1,39 @@
 language: python
-addons:
-    firefox: "52.0esr"
+dist: xenial  # required for Python >= 3.7 (travis-ci/travis-ci#9069)
 python:
     - 2.7
-before_install:
-    - "export DISPLAY=:99"
-    - "sh -e /etc/init.d/xvfb start"
+    - 3.6
+    - 3.7
+env:
+    - TOXENV=django111
+    - TOXENV=django20
+    - TOXENV=django21
+matrix:
+  include:
+    - python: 2.7
```

Comment:
You need **python: 3.6** here because of [this tox.ini line](https://github.com/edx/xblock-utils/pull/56/files#diff-b91f3d5bd63fcd17221b267e851608e8R36)